### PR TITLE
*: add more cop task and memory info to slow_query table

### PR DESCRIPTION
**Pull request by crazycs520**
### What problem does this PR solve? 
1. add more cop task and memory info to slow_query table.

### What is changed and how it works?
1. Change `slow_query` table info and chage slow log parser to parser more field.

### Check List 

Tests 

 - Unit test

Code changes

 - Has exported function/method change

Side effects

Related changes

 - Need to cherry-pick to the release branch

### CODE REVIEW

File: infoschema/slow_log.go [hunk 1]
```diff
@@ -125,23 +135,33 @@ func ParseSlowLog(tz *time.Location, scanner *bufio.Scanner) ([][]types.Datum, e
...
+	indexIDs          string
+	isInternal        bool
+	digest            string
+	statsInfo         string
+	numCopTasks       uint64
+	avgProcessTime    float64
+	p90ProcessTime    float64
+	maxProcessTime    float64
+	maxProcessAddress string
+	AvgWaitTime       float64
```
Comment (bb7133) [inline infoschema/slow_log.go]:
```suggestion
	avgWaitTime       float64
```
ditto for variables below

**Comment by bb7133** [comment]
LGTM

File: infoschema/slow_log.go (hunk 1 above)
Comment (crazycs520) [reply infoschema/slow_log.go]: done. PTAL

**Comment by zz-jason** [comment]
@crazycs520 please fix ci

**Pull request merged**